config.json

```
@@ -1,5 +1,6 @@
 {
   "site_name": "Warga Bantu Warga",
   "site_tagline": "Informasi Faskes & Alkes untuk COVID-19",
-  "site_description": "Inisiatif warga untuk berbagi informasi seputar fasilitas kesehatan & alat kesehatan untuk COVID-19. WargaBantuWarga memudahkan publik mencari & mengakses informasi."
+  "site_description": "Inisiatif warga untuk berbagi informasi seputar fasilitas kesehatan & alat kesehatan untuk COVID-19. WargaBantuWarga memudahkan publik mencari & mengakses informasi.",
+  "site_url": "https://wargabantuwarga.com"
```

This config file is managed by Netlify CMS as well, so please add it into the `public/admin/config.yml` configuration  to avoid accidental removal.

And the official URL is with `www.` prefix to avoid potential issues caused by the apex domain.
```suggestion
  "site_url": "https://www.wargabantuwarga.com"
```